proof/access-control/ARM/ArchAccess.thy

```diff
@@ -138,6 +143,7 @@ abbreviation integrity_asids_aux ::
      pp_opt = pp_opt' \<or> (\<forall>asid'. asid' \<noteq> 0 \<and> asid_high_bits_of asid' = asid_high_bits_of asid
                                  \<longrightarrow> pasASIDAbs aag asid' \<in> subjects)"
 
+\<comment> \<open>FIXME: could we define integrity_asids to operate on arch_state only?\<close>
```

Comment:
Unfortunately not, we use `asid_pools_of` in RISCV64

Reply:
Ah, that makes sense, I was only looking at ARM. We could probably make this a two layer definition with projections so that we have a nicer internal version within arch files while maintaining the interface, but I'm not sure that that is worthwhile.